extension/src/validator/validator-builder.js

```diff
@@ -6,8 +6,21 @@ function withPayment(paymentObject) {
   const errors = {}
 
   return {
+    validateMetadataFields() {
+      if (!paymentObject.custom && !paymentObject.custom.fields) return this
+      if (!paymentObject.custom.fields.commercetoolsProjectKey ||
+          paymentObject.custom.fields.commercetoolsProjectKey === ' ')
+        errors.missingRequiredCtpProjectKey =
+            errorMessages.MISSING_REQUIRED_FIELDS_CTP_PROJECT_KEY
+      if (!paymentObject.custom.fields.adyenMerchantAccount ||
+          paymentObject.custom.fields.adyenMerchantAccount === ' ')
```

Comment:
```suggestion
          paymentObject.custom.fields.adyenMerchantAccount.trim() === '')
```

Reply:
Changed to check if the field contains any whitespace 9f61a99